schema/notemons.py

```diff
@@ -0,0 +1,17 @@
+QSChapter = {
+    "section_id" : "s",
+    "section_name" : "t",
+    "short_1" : "s",
+    "short_2" : "s"
```

Comment:
It looks like there is only 1 short after the name, so the 2nd short is not needed

- [ ] Remove short_2

Reply:
My mistake, will fix.